chibuezeayogu/React-Math-Skills-Game
# #160742676 Setup webpack and babel

**What does the PR do?**
- Add `Webpack and Babel ` configuration

**Description of Task to be completed?**
- Ensure `Webpack and Babel ` is properly configured 

**How should this be manually tested?**
- Clone the repository [here](https://github.com/chibuezeayogu/React-Math-Skills-Game.gitt)
-  cd into `React-Math-Skills-Game` and checkout to this branch `chore-setup-webpack-and-babel-160742676`
- run `npm start` webpack should compile successfully 

**What are the relevant pivotal tracker stories?**
- [#160742719](https://www.pivotaltracker.com/story/show/160742719)

**Screenshots**
